criextension/criextension.proto

```diff
@@ -0,0 +1,30 @@
+syntax = 'proto3';
+
+package criextension.api;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto";
+
+option go_package = "criextension";
+
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all) =  true;
+option (gogoproto.goproto_getters_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unrecognized_all) = false;
+
+service CRIExtensionRuntimeService {
+    rpc UpdateContainerResourcesV2(UpdateContainerResourcesV2Request) returns (UpdateContainerResourcesV2Response) {}
+}
+
+message UpdateContainerResourcesV2Request {
```

Comment:
Maybe I am missing something here but now that the other [PR in Kubernetes](https://github.com/kubernetes/kubernetes/pull/95741/) is merged and includes `WindowsContainerResources` in the `UpdateContainerResourceRequest` can we use that?

Reply:
+1. I'm confused as well :)

Reply:
I think we thought it might've taken longer to get those changes in than it did but @katiewasnothere your commits actually already in the standalone cri repo also https://github.com/kubernetes/cri-api/tree/master/pkg/apis/runtime/v1alpha2

Reply:
I was under the impression that we would wait for a tagged version of cri-api with the changes I made in them, but talking to Kevin and based on what y'all said, I think it would be fine to take a commit version instead. In which case I think this PR is obsolete :p